tests/tests_ovirt_elasticsearch.yml

```diff
@@ -0,0 +1,55 @@
+- name: Ensure that the role runs with parameters from ovirt to two elasticsearch outputs and local files output
+  hosts: all
+  become: true
+
+  tasks:
+    - name: deploy config to send to elasticsearch
+      vars:
+        logging_purge_confs: true
+        logging_outputs:
+          - name: elasticsearch_output
+            type: elasticsearch
+            server_host: logging-es
+            server_port: 9200
+            index_prefix: project.
+            input_type: ovirt
+            retryfailures: off
+            ca_cert: "/etc/rsyslog.d/es-ca.crt"
+            cert: "/etc/rsyslog.d/es-cert.pem"
+            key: "/etc/rsyslog.d/es-key.pem"
+          - name: elasticsearch_output_ops
+            type: elasticsearch
+            server_host: logging-es-ops
+            server_port: 9200
+            index_prefix: .operations.
+            input_type: viaq
+            retryfailures: off
+            ca_cert: "/etc/rsyslog.d/es-ca.crt"
+            cert: "/etc/rsyslog.d/es-cert.pem"
+            key: "/etc/rsyslog.d/es-key.pem"
+        logging_inputs:
+          - name: ovirt_input
+            type: ovirt
+            collect_ovirt_engine_log: true
+            collect_ovirt_vdsm_log: false
```

Comment:
where are `collect_ovirt_engine_log` and `collect_ovirt_vdsm_log` used?

Reply:
> where are `collect_ovirt_engine_log` and `collect_ovirt_vdsm_log` used?

I made them use in rsyslog_conf_ovirt_formatting in the ovirt vars main.yml. In this commit, I'm setting the default values, as well. To complete the changes, I need the default values of ovirt_env_uuid_logs and ovirt_env_uuid_metrics. > @sradco.
https://github.com/linux-system-roles/logging/pull/113/commits/8914fcf8bebcc387f268c61f6326533806fc5ddf

Reply:
>  need the default values of ovirt_env_uuid_logs and ovirt_env_uuid_metrics.

If these are the OpenShift uuid of the OpenShift namespace used for ovirt logs and ovirt metrics, then there isn't a default value.

Reply:
All right. Thanks! I'm updating the code based on it.